addons/im_livechat/views/im_livechat_channel_templates.xml

```diff
@@ -185,6 +185,7 @@
             <script type="text/javascript" src="/mail/static/src/js/utils.js"></script>
             <script type="text/javascript" src="/im_livechat/static/src/legacy/public_livechat.js"></script>
             <t t-call="web._assets_helpers"/>
+            <link rel="stylesheet" type="text/css" href="/web/static/lib/fontawesome/css/font-awesome.css"/>
```

Comment:
Not sure about that one.
Isn't the goal of external livechat assets to be as tiny as possible to load quickly on external websites?

Do we really need the whole font?
The file is pretty big (2000 lines) and also loads fonts with relative links.

Reply:
Indeed, this can also conflict with whatever font-awesome version is potentially installed on the embedding website.

Reply:
Correct!